os/dutils.sh

```diff
@@ -57,6 +60,58 @@ function FIND_DBGBINFILE()
 	KERNEL_DBG_BIN="tinyara$EXEEXT"
 }
 
+function TRAP_MENU()
+{
+	echo ======================================================
+	echo "  \"Run TRAP tool for\""
+	echo ======================================================
+	echo "  \"1. Existing binary\""
+	echo "  \"2. New binary\""
+	echo "  \"x. Exit\""
+	echo ======================================================
+
+	read SELECT_BIN
+	case ${SELECT_BIN,,} in
+		1|default)
+			if [ -z "$(ls -A $BINDIR)" ]; then
+				echo "No output file, Build the code and run TRAP again."
+				exit 1
+			fi
+			echo "Enter the name of the crash log file:"
+			read LOG
+			cp $LOG $TRAPDIR
+			TRAP_RUN log $LOG
+			;;
+		2|manual)
+			echo "Enter the following wrt os directory"
+			echo "Enter the binary folder name:"
+			echo "Warning: Current build files in /build/output/bin will be overwritten."
```

Comment:
Is this necessary to overwrite? How about modifying the TRAP to get bin path and log file path?
If anyone is not given, use default value.

Reply:
Updated